mm2src/coins/lp_coins.rs

```diff
@@ -2452,6 +2452,10 @@ impl CoinBalance {
         }
     }
 
+    pub fn new_with_unspendable(spendable: BigDecimal, unspendable: BigDecimal) -> CoinBalance {
+        CoinBalance { spendable, unspendable }
+    }
```

Comment:
Just my opinion, not a game-changer blocker:

It doesn't feel right to have these functions as `CoinBalance` fields are already `pub`ed and the use of

```rs
let _ = CoinBalance {
    spendable,
    unspendable
};
```

seems more clean than

```rs
CoinBalance::new_with_unspendable(x, y)
```

because we can't know what is the implementation detail behind `new_with_unspendable`.